src/data/poolList.ts

```diff
@@ -234,120 +234,150 @@ export const poolList: Record<KnownNetwork, StaticPoolInfo[]> = {
         {
             // 1x
             name: '1-BTC/USDC',
-            address: '0x1b9A08Bd1b976fd91a355625509FbFcbbF34fb20',
+            address: '0x4eDc3814959705E59afeabbb52117950333c0721',
             leverage: 1,
             updateInterval: FIVE_MINUTES,
             frontRunningInterval: THIRTY_SECONDS,
```

Comment:
these are all one hour upkeep interval + 5 minute frontrunning interval for the current deploy